.circleci/config.yml

```diff
@@ -37,6 +37,9 @@ workflows:
           npm-run-command: build
           setup:
             - run: node --version
+            - run:
+                name: Set ENABLE_PISA
+                command: echo 'export ENABLE_PISA=true' >> $BASH_ENV
```

Comment:
* [ ] FYI setting the job env could be cleaner: https://circleci.com/docs/set-environment-variable/#set-an-environment-variable-in-a-job

Reply:
See https://github.com/Scrivito/scrivito-portal-app/pull/717